pdb-alignment-api/src/main/java/org/cbioportal/pdb_annotation/web/domain/EnsemblRepository.java

```diff
@@ -12,6 +11,6 @@
  *
  */
 @Transactional
-public interface EnsemblRepository extends JpaRepository<Alignment, Long> {
+public interface EnsemblRepository extends JpaRepository<Ensembl, Long> {
```

Comment:
It is a good practice to set the repository name same as the model name, thanks for the modification @sheridancbio!